modelzoo/resnet50.py

```diff
@@ -0,0 +1,192 @@
+import lasagne
```

Comment:
Please add a link to the model and the license at the beginning of the modelzoo file, as in the other modelzoo/*.py files (e.g., https://github.com/Lasagne/Recipes/blob/master/modelzoo/vgg16.py).

In the notebook, you might mention at the end of the introduction that there's a pre-trained resnet50 in the model zoo, and that the notebook explains how to reproduce it. The pickle file doesn't need to be downloaded for the notebook at all, does it?